vignettes/modification.Rmd

```diff
@@ -119,6 +126,29 @@ xml_add_child(t1, read_xml("<test/>"))
 x
 ```
 
+## Removing nodes ##
+The `xml_remove_node()` can be used to remove a node (and it's children) from a
+tree. The default behavior is to unlink the node from the tree, but does not
+free the memory for the node, so R objects pointing to the node are still
+valid. This allows code like the following to work without crashing R
```

Comment:
Technically I guess it is only a leak if there are no longer pointers to the memory which is not the case here but I will try to clarify.